stdlib/public/TensorFlow/Ops.swift

```diff
@@ -577,6 +577,38 @@ public extension Tensor where Scalar : FloatingPoint & Equatable {
   }
 }
 
+public extension Tensor where Scalar : FloatingPoint {
+  // TODO: std() should handle non floating point Tensors.
+
+  /// Returns the standard deviation along the specified axes. The reduced
+  /// dimensions are retained with value 1. Does not apply Bessel's correction.
+  /// - Parameter axes: The dimensions to reduce.
+  /// - Precondition: Each value in `axes` must be in the range `-rank..<rank`.
+  @differentiable(wrt: self where Scalar : TensorFlowFloatingPoint)
+  func std() -> Tensor {
```

Comment:
I think I would prefer the full name `func standardDeviation`, despite `std` being a term-of-art and `standardDeviation(alongAxes:)` being really long.

I searched for usages of `std` and I don't think it's a super commonly used operation.

What do you think? @saeta @rxwei

Reply:
Another reason in favor of `standardDeviation`: abbreviated names like `std` and `stddev` look a bit weird with the `alongAxes:` label.

```swift
x.std()
x.stddev()
x.standardDeviation()

x.std(alongAxes: 0)
x.stddev(alongAxes: 0)
x.standardDeviation(alongAxes: 0)
```

Reply:
Completely agreed.

Reply:
I've now updated the name. Thanks!